make/tools.mk

```diff
@@ -109,6 +109,13 @@ vendor-go: bin/tools/go
 unvendor-go: bin/tools/go
 	rm -rf bin/tools/go bin/tools/goroot
 
+.PHONY: which-go
+## Print the version and path of go which will be used for building and
+## testing in Makefile commands. Vendored go will have a path in ./bin
+which-go: |  $(DEPENDS_ON_GO)
+	@$(GO) version
+	@echo "go binary used for above version information: $(GO)"
```

Comment:
note: this makes it so much easier to see at a glance what version of Go the _makefile_ is gonna use!